src/mrack/utils.py

```diff
@@ -201,6 +206,51 @@ def get_ssh_key(host, meta_host, config):
     return ssh_key
 
 
+def ssh_to_host(
+    host,
+    execute=None,
```

Comment:
IMHO `command` is a better name, also I think we should pass this as last argument.